CMakeLists.txt

```diff
@@ -42,7 +45,7 @@ if(APPLE)
 endif()
 if(LINUX)
   set(sys_libs pthread dl rt)
-  set(cxx_flags "-Werror=old-style-cast -Werror -Wall -Wextra -Winit-self -Wreturn-type -Wunused-variable -Wsign-compare -Warray-bounds -Wunknown-pragmas -Wuninitialized -Wstrict-aliasing -Wunused-value -Wunused-label -Wswitch -Wcast-align -Wno-ctor-dtor-privacy -Wno-missing-noreturn -Wno-unused-parameter -Werror=old-style-cast -Wreorder")
+  set(cxx_flags "-Werror -Wall -Wextra -Winit-self -Wreturn-type -Wunused-variable -Wsign-compare -Warray-bounds -Wunknown-pragmas -Wuninitialized -Wstrict-aliasing -Wunused-value -Wunused-label -Wswitch -Wcast-align -Wno-ctor-dtor-privacy -Wno-missing-noreturn -Wno-unused-parameter -Wreorder")
```

Comment:
I'm not sure we can remove this.  The "old style cast" rejection was added recently for a user who has that flag turned on in their project (and so needs hobbes to have it on as well in order to embed there).

I had to go through a massive rewriting of hobbes code a while back to rewrite every C-style cast to an appropriate cast.